GTG/gtk/editor/editor.py

```diff
@@ -60,30 +65,30 @@ def __init__(self,
         self.clipboard = clipboard
         self.builder = Gtk.Builder()
```

Comment:
Taking this back, that breaks the assumptions from plugins API quite horribly :/